bin/logstash.lib.sh

```diff
@@ -15,19 +15,34 @@ setup_java() {
   fi
 
   if [ ! -x "$JAVACMD" ] ; then
-    echo "Could not find any executable java binary. Please install java in your PATH or set JAVA_HOME."
+    echo "Could not find any executable java binary. Please install java in your PATH or set JAVA_HOME." 1>&2
     exit 1
   fi
 
-  JAVA_OPTS="$JAVA_OPTS -Xmx${LS_HEAP_SIZE}"
-  JAVA_OPTS="$JAVA_OPTS -XX:+UseParNewGC"
-  JAVA_OPTS="$JAVA_OPTS -XX:+UseConcMarkSweepGC"
-  JAVA_OPTS="$JAVA_OPTS -Djava.awt.headless=true"
+  if [ "$JAVA_OPTS" ] ; then
+    echo "WARNING: Default JAVA_OPTS are going to be overridden by the system ones. New JAVA_OPTS are $JAVA_OPTS"  1>&2
```

Comment:
I still don't like too much the working, but that's cosmetic. some suggestions:

"are going to" -> "will be"

"by the system ones" -> "by JAVA_OPTS defined in the environment"

"New JAVA_OPTS are" -> "environment JAVA_OPTS are"